frontend/app/components/topnav.hbs

```diff
@@ -88,6 +88,18 @@
       </Topnav::List>
       <Topnav::List class="md:ml-auto md:border-t-0">
         <ReportReviewWarning @class="max-md:hidden" />
+        <Topnav::ListItem>
+          <Topnav::LinkTo
+            @onClick={{fn (mut this.expand) false}}
+            title="Timed Documentation Of This Page"
+            href={{this.docs.docsEndpoint}}
+            target="_blank"
+            data-test-docs-link
+          >
+            <FaIcon @icon="info-circle" @prefix="fa" />
```

Comment:
Would be nice to have a preview for MRs changing the app UI. For example: Not sure if the `info-circle` icon is looking good. I could imagine `circle-question` may be suited better.

Reply:
Can I get a preview? :point_right: :point_left: *pls*

Reply:
Regarding https://github.com/adfinis/timed/pull/570#issuecomment-2792105628
 I think we don't need the `Docs` title for the link. The icon + hover text should be enough. We don't want to overload the top nav to much.